### ISSUE TITLE
20:9 support

### ISSUE THREAD

**Issue by Envel-Nikita-Gutsenkov** [TeamNewPipe/NewPipe#7338]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### What feature do you want?
If I view the video, then everything is ok. if I go to the video list in horizontal mode, then I have a black bar due to the fact that the application does not support new display formats, for example, I have 20:9

#### Why do you want this feature?
displays on the entire display

#### Screenshot to illustrate problem

https://i.imgur.com/RP841cL.jpg

**Comment by litetex** [comment]
Duplicate of #7211

**Comment by Envel-Nikita-Gutsenkov** [comment]
> Duplicate of #7211

no, this does not apply to the topic you have attached. this only applies to the list of videos in the app

**Comment by Envel-Nikita-Gutsenkov** [comment]
> Duplicate of #7211

https://imgur.com/RP841cL

**Comment by opusforlife2** [comment]
@Envel-Nikita-Gutsenkov To avoid confusion like this, it is best to upload a screenshot when opening the issue. I've edited your issue and added the screenshot you've provided.

**Comment by litetex** [comment]
Can't reproduce the problem on a 20:9 emulated device:
![grafik](https://user-images.githubusercontent.com/40789489/140802052-67598502-6245-4350-ae69-1ef87d7da198.png)


Do you have some kind of notch option enable which maybe crops the app?
Information about the device/OS would also be useful.

**Comment by Envel-Nikita-Gutsenkov** [comment]
Yes, camera, xiaomi 11 lite, MIUI 12

but I don't think this is a reason to trim the interface, since the camera is not significant.

**Comment by litetex** [comment]
> but I don't think this is a reason to trim the interface, since the camera is not significant.

I don't know what that means or how it's related to my above question.

Also again:
Do you have some kind of notch option enabled (in the OS) which maybe crops the app?

My smartphone e.g. [has options to "hide the notch" and how apps should behave when they encounter one (full-screen)](https://consumer.huawei.com/en/support/content/en-us00753709/)

Do other apps also have the same problem?

**Comment by Envel-Nikita-Gutsenkov** [comment]
> > but I don't think this is a reason to trim the interface, since the camera is not significant.
> 
> I don't know what that means or how it's related to my above question.
> 
> Also again: Do you have some kind of notch option enabled (in the OS) which maybe crops the app?

No
> 
> My smartphone e.g. [has options to "hide the notch" and how apps should behave when they encounter one (full-screen)](https://consumer.huawei.com/en/support/content/en-us00753709/)
> 
> Do other apps also have the same problem?
No.

**Comment by busssard** [comment]
I have the problem now that the player cuts away video in full screen mode. As my phone is too wide(20:9)
I would like to be able to enable/disable the fitting. So that i either have black bars or cut some video away on my own choosing.
